soh/src/overlays/actors/ovl_En_Mag/z_en_mag.h

```diff
@@ -41,10 +41,7 @@ typedef enum {
     /* 0x04 */ MAG_STATE_POST_DISPLAY
 } EnMagGlobalState;
 
-#define dgTitleCopyrightGCTex "__OTR__objects/object_mag/gTitleCopyright19982003Tex"
-static const ALIGN_ASSET(2) char gTitleCopyrightGCTex[] = dgTitleCopyrightGCTex;
-
-#define dgTitleCopyrightN64Tex "__OTR__objects/object_mag/gTitleCopyright1998Tex"
-static const ALIGN_ASSET(2) char gTitleCopyrightN64Tex[] = dgTitleCopyrightN64Tex;
+#define dgTitleCopyright1998Tex "__OTR__objects/object_mag/gTitleCopyright1998Tex"
+static const ALIGN_ASSET(2) char gTitleCopyright1998Tex[] = dgTitleCopyright1998Tex;
```

Comment:
`gTitleCopyright19982003Tex` needs a definition and `static const ALIGN_ASSET(2)` declaration here just like `gTitleCopyright1998Tex`. The ALIGN_ASSET part fixes a crash on some platforms. Or did it already have one?

Reply:
`gTitleCopyright19982003Tex` is already defined in our generated headers with the ALIGN_ASSET part.

Reply:
The 19982003 one comes from the extract header step

Reply:
Right, I didn't see it in the diff (which makes sense) so I wanted to make sure that had been checked, thanks!